_process_utilities.py

```diff
@@ -75,68 +60,113 @@
 sentinelStack.processingMethod            = auto         # [sbas, squeesar, ps]
 sentinelStack.demMethod                   = auto         # [bbox, ssara]
 '''
+#### More options to be added if required:
+#sentinelStack.pairList                    = auto         # [file] file containing pairs to process in each line. auto for None
+#sentinelStack.overridePairs               = auto         # [yes, no] override program-generated pairs with the files on the list. auto for no
+#sentinelStack.cleanUp                     = auto         # [False, True] Remove fine*int burst fines. auto for False
+#sentinelStack.layoverMask                 = auto         # [False, True] Generate layover mask and remove from filtered phase. auto for False
+#sentinelStack.waterMask                   = auto         # [False, True] Generate water mask and remove from filtered phase. auto for False
+#sentinelStack.virtualFiles                = auto         # [False, True] writing only vrt of merged files (Default: True)
+#sentinelStack.forceOverride               = auto         # Force new acquisition override. auto for no
 
 
 ##########################################################################
 
 
-def set_inps_value_from_template(inps, template_key,
-                                 inps_name, default_name='auto',
-                                 default_value=None, REQUIRED=False):
-    """
-    Processes a template parameter and adds it to the inps namespace.
-    Options for setting both default values and required parameters
-    :param inps: The parsed input namespace
-    :param template: The parsed dictionary (namespace)
-    :param template_key: The desired key in `template`
-    :param inps_name: The parameter name to assign in `inps`
-    :param default_name: 'auto' is the normal placeholder
-    :param default_value: Default value to assign in `inps`
-    :param REQUIRED: Throws error if REQUIRED is True
-    :return: None
-    """
+def send_logger():
+    return logger
 
-    # Allows you to refer to and modify `inps` values
-    inps_dict = vars(inps)
+##########################################################################
 
```

Comment:
Skip this part, it is mostly rearranging functions. some of the useless functions are removed and some of them moved to new scripts.(create_stacksentinel_run_files.py, execute_stacksentinel_run_files.py, ingest_insarmaps.py)